packages/@sanity/form-builder/src/sanity/uploads/uploadImage.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import Observable from '@sanity/observable'
+import readExif from './image/readExif'
+import rotateImage from './image/rotateImage'
+import {DEFAULT_ORIENTATION} from './image/orient'
+import {set} from '../../utils/patches'
+import type {UploadEvent} from './typedefs'
+// The eslint import plugin doesn't work well with opaque types
+// https://github.com/benmosher/eslint-plugin-import/issues/921
+// https://github.com/gajus/eslint-plugin-flowtype/issues/260
+// eslint-disable-next-line import/named
+import type {OrientationId} from './image/orient'
+import type {ObservableI} from '../../typedefs/observable'
+import {UPLOAD_STATUS_KEY} from './constants'
+import {CLEANUP_EVENT, INIT_EVENT, createUploadEvent} from './utils'
+import {createUploadId, getUploadEvents, scheduleUpload} from './uploadQueue'
+
+type Exif = {
+  orientation: OrientationId
+}
+
+const setInitialUploadState$ = Observable.of(INIT_EVENT)
+const unsetUploadState$ = Observable.of(CLEANUP_EVENT)
+
```

Comment:
Should this be removed?

Reply:
Yeah, I forgot to delete it, just wanted to keep it around for a little while after testing concurrency yesterday. Thanks for the heads up 😅